### PR TITLE
[ALLUXIO-2262] Mark nested class Name in PropertyKey with annotation "@ThreadSafe"

### PR DESCRIPTION
**Pull request by schernolyas**
https://alluxio.atlassian.net/browse/ALLUXIO-2262

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import javax.annotation.concurrent.ThreadSafe;
```
Comment (apc999) [inline core/common/src/main/java/alluxio/PropertyKey.java]: Add an empty line between line 15 and 16?
`javax.*` imports should be separated with `java.*` imports by an empty line.
http://www.alluxio.org/docs/master/en/Contributing-to-Alluxio.html#coding-style

**Comment by gpang** [comment]
Jenkins, test this please

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 1 above)
Comment (schernolyas) [inline core/common/src/main/java/alluxio/PropertyKey.java]: Hi!

Fixed. Recheck please

**Comment by schernolyas** [comment]
Hi @apc999 !

Why You now merge the PR?

**Pull request merged**